x-pack/plugin/esql/src/main/antlr/EsqlBaseLexer.g4

```diff
@@ -638,3 +640,15 @@ CLOSING_METRICS_BY
 CLOSING_METRICS_PIPE
     : PIPE -> type(PIPE), popMode
     ;
+
+//
+// Fork
+//
+mode FORK_MODE;
+// double push since expression mode double pops
+FORK_OPENING_BRACKET : OPENING_BRACKET -> type(OPENING_BRACKET), pushMode(DEFAULT_MODE);
+FORK_CLOSING_BRACKET : CLOSING_BRACKET -> type(CLOSING_BRACKET), popMode;
```

Comment:
Use `()` instead of `[]` - after spending more time researching subqueries, normal parenthesis work better in the language design than square brackets, hence the plan to use the former for subqueries as well.

Reply:
done.